pkgs/development/tools/analysis/hopper/default.nix

```diff
@@ -1,42 +1,57 @@
-{ stdenv, fetchurl, pkgs, lib }:
+{
+  stdenv, fetchurl, lib,
 
-stdenv.mkDerivation rec {
+  autoPatchelfHook,
+
+  wrapQtAppsHook,
+
+  libbsd,
+  python27,
+  gmpxx,
+}:
+let
+
+in stdenv.mkDerivation rec {
   pname    = "hopper";
-  version = "4.5.7";
+  version = "4.5.19";
   rev = "v${lib.versions.major version}";
 
   src = fetchurl {
     url = "https://d2ap6ypl1xbe4k.cloudfront.net/Hopper-${rev}-${version}-Linux.pkg.tar.xz";
-    sha256 = "1ce7a0f13126a940398aa8da4a74e250dff0401074f30446a8840ac3dbb902c0";
+    sha256 = "1c9wbjwz5xn0skz2a1wpxyx78hhrm8vcbpzagsg4wwnyblap59db";
   };
 
   sourceRoot = ".";
 
-  ldLibraryPath = with pkgs; stdenv.lib.makeLibraryPath  [
-libbsd.out libffi.out gmpxx.out python27Full.out python27Packages.libxml2 qt5.qtbase zlib  xlibs.libX11.out xorg_sys_opengl.out xlibs.libXrender.out gcc-unwrapped.lib
+  nativeBuildInputs = [
+    wrapQtAppsHook
+    autoPatchelfHook
   ];
 
-  nativeBuildInputs = [ pkgs.qt5.wrapQtAppsHook ];
-
-  qtWrapperArgs = [ ''--suffix LD_LIBRARY_PATH : ${ldLibraryPath}'' ];
+  buildInputs = [
+    libbsd
+    python27
+    gmpxx
+  ];
 
   installPhase = ''
     mkdir -p $out/bin
     mkdir -p $out/lib
     mkdir -p $out/share
+
     cp $sourceRoot/opt/hopper-${rev}/bin/Hopper $out/bin/hopper
     cp -r $sourceRoot/opt/hopper-${rev}/lib $out
     cp -r $sourceRoot/usr/share $out/share
```

Comment:
This is installing files into `$out/share/share`

Reply:
done